src/Control/Monad/Error/Class.purs

```diff
@@ -63,6 +63,26 @@ try
   -> m (Either e a)
 try a = (Right <$> a) `catchError` (pure <<< Left)
 
+-- | Takes a default and a `Maybe` value. If the value is a `Nothing` throw the
+-- | default value as an error, otherwise lift the `Just` value into `m`.
+-- |
+-- | ```purescript
+-- | note "default" Nothing = throwError "default"
+-- | note "default" (Just 1) = pure 1
+-- | ```
+note :: forall e m. MonadThrow e m => e -> Maybe ~> m
+note e = maybe (throwError e) pure
+
+-- | Absorbs exceptions in an `m a`, turning excepted values into `Nothing` and
+-- | wrapping succesful values in a `Just`
+-- |
+-- | ```purescript
+-- | hush (throwError "ParseError") = pure Nothing
+-- | hush (pure 42) = pure (Just 42)
+-- | ```
+hush :: forall e m a. MonadError e m => m a -> m (Maybe a)
+hush ma = catchError (Just <$> ma) (const (pure Nothing))
```

Comment:
This is also `map Either.hush <<< try` I think, which I think is short enough to probably not need its own function. Also, it's not really the same as `Either.hush` because of the extra `m`, so if we do keep it, maybe it should have a slightly different name.